H/HiGHS/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "HiGHS"
-version = v"0.2.2"
 
-# Collection of sources required to complete build
+version = v"0.2.4"
```

Comment:
```suggestion
version = v"0.2.3"
```

Reply:
That was because of my tests locally, 0.2.3 is good